src/lib.rs

```diff
@@ -98,6 +99,26 @@ impl PGAppend<*mut i8> for pg::StringInfo {
     }
 }
 
+struct Wrapped(pg::Enum_ReorderBufferChangeType);
+
+impl fmt::Display for Wrapped {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        use pg::Enum_ReorderBufferChangeType::*;
+        let formatted_token = match self.0 {
+            REORDER_BUFFER_CHANGE_INSERT => "insert",
+            REORDER_BUFFER_CHANGE_UPDATE => "update",
+            REORDER_BUFFER_CHANGE_DELETE => "delete",
+            REORDER_BUFFER_CHANGE_INTERNAL_SNAPSHOT => "internal_snapshot",
+            REORDER_BUFFER_CHANGE_INTERNAL_COMMAND_ID => "internal_command_id",
+            REORDER_BUFFER_CHANGE_INTERNAL_TUPLECID => "internal_tuplecid",
+            REORDER_BUFFER_CHANGE_INTERNAL_SPEC_INSERT => "internal_spec_insert",
+            REORDER_BUFFER_CHANGE_INTERNAL_SPEC_CONFIRM => "internal_spec_confirm",
+            REORDER_BUFFER_CHANGE_MESSAGE => "change_message"
```

Comment:
So `REORDER_BUFFER_CHANGE_MESSAGE` is the catch all? Maybe it would be better to have a future proof `_`?